queries.sql

```diff
@@ -18,3 +18,81 @@ SELECT * FROM animals WHERE name!='Gabumon';
 
 SELECT * FROM animals WHERE
 weight_kg>=10.4 and weight_kg<=17.3;
+
+-- Day 2. Query and Update Animals Table
+BEGIN TRANSACTION;
+
+UPDATE animals
+SET species = 'unspecified';
+
+SELECT species FROM animals;
+
+ROLLBACK TRANSACTION;
+
+SELECT species FROM animals;
+
+BEGIN TRANSACTION;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE Name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT TRANSACTION;
+
+SELECT species FROM animals;
+
+BEGIN TRANSACTION;
+
+DELETE FROM animals;
+
+ROLLBACK TRANSACTION;
+
+SELECT * FROM animals;
+
+BEGIN TRANSACTION;
+
+DELETE FROM animals
+WHERE date_of_birth > '2022-01-01';
+
+SELECT * FROM animals;
+
+SAVEPOINT delete_young;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+
+SELECT weight_kg FROM animals;
+
+ROLLBACK TO delete_young;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+COMMIT TRANSACTION;
+
+SELECT weight_kg FROM animals;
+
+SELECT COUNT(*) FROM animals;
+
+SELECT COUNT(escape_attempts) FROM animals
+WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg) FROM animals;
+
+SELECT neutered, SUM(escape_attempts)
+FROM animals
+GROUP BY neutered;
+
```

Comment:
- [ ] Kindly make sure this query answers correctly to the question asked. You are supposed to get animals with the most `escape_attempts`, neutered or not neutered animals, and not the `sum` of `escape_attempts` on each type. To fix the issue you can just edit your query where you use `SUM(escape_attempts)` to have `MAX(escape_attempts)` or `AVG(escape_attempts)`(both of these approaches will give you the same result).

- [ ] Once you make the change above make sure you take the screenshot result and update the screenshot from the pull request description with the correct one. Good job and happy coding :tada: